go/engine/identify2_test.go

```diff
@@ -1022,7 +1022,7 @@ func _testNoSelfHostedIdentifyInPassiveMode(t *testing.T, sigVersion libkb.SigVe
 	defer tc.Cleanup()
 
 	eve := CreateAndSignupFakeUser(tc, "e")
-	_, _, err := proveRooter(tc.G, eve)
+	_, _, err := proveRooter(tc.G, eve, sigVersion)
```

Comment:
Was this test running twice before, but just re-testing v1 both times?

Reply:
yeah